modules/programs/ssh.nix

```diff
@@ -118,12 +137,14 @@ let
   matchBlockStr = cf: concatStringsSep "\n" (
     ["Host ${cf.host}"]
     ++ optional (cf.port != null)         "  Port ${toString cf.port}"
+    ++ optional (cf.forwardAgent != null) "  ForwardAgent ${yn cf.forwardAgent}"
     ++ optional cf.forwardX11             "  ForwardX11 yes"
     ++ optional cf.forwardX11Trusted      "  ForwardX11Trusted yes"
     ++ optional cf.identitiesOnly         "  IdentitiesOnly yes"
     ++ optional (cf.user != null)         "  User ${cf.user}"
     ++ optional (cf.identityFile != null) "  IdentityFile ${cf.identityFile}"
     ++ optional (cf.hostname != null)     "  HostName ${cf.hostname}"
+    ++ optional (length cf.sendEnv > 0)   "  SendEnv ${join " " cf.sendEnv}"
```

Comment:
Instead of `(length cf.sendEnv > 0)` use `cf.sendEnv != []`.

Reply:
fixed in 7e1f483